### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dginelli**
Updated the README.md file with the information about the data contained in the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,255 @@
...
+| java.lang.IllegalArgumentException   |         420 |
+| java.io.IOException                  |         404 |
+| **Other**                            |       2.414 |
+| **Total**                            |      18.725 |
+
+On the contrary, taking into account only the custom exceptions, the top 10 common failure types observed are the following ones:
+
+| Failure Type                                            | Occurrences |
+|:--------------------------------------------------------|------------:|
+| skipped                                                 |         346 |
```
Comment (monperrus) [inline README.md]: what is the corresponding exception?

File: README.md [hunk 2]
```diff
@@ -1,25 +1,255 @@
...
+| **6**                                      |                   735  |
+| **&gt; 6**                                 |                   956  |
+
+The highest number of the failing test cases observed in a single build has been 804 (`build` [412170129](https://github.com/repairnator/repairnator-experiments/blob/master/builds/412170129.json), `branch` [SeleniumHQ-htmlunit-driver-412170129-20180804-232323](https://github.com/repairnator/repairnator-experiments/tree/SeleniumHQ-htmlunit-driver-412170129-20180804-232323)).
+
+### Number of erroring test cases per build
+
+The number of the builds based on the number of their erroring test cases is reported in the following table:
+
+| Number of the erroring test cases          | Number of the builds   |
```
Comment (monperrus) [inline README.md:337]: it's never 0?

File: README.md (hunk 1 above)
Comment (dginelli) [reply README.md]: This was indeed a point that I wanted to write in the previous email, but I forgot to cite it. Looking at the repairnator.json files, there are some of them in which there is the voice "skipped" as one of the values of the array "FailureNames" contained in the object "metrics". This is an example: https://github.com/repairnator/repairnator-experiments/blob/70a4fbb5517facbcc39621a3ca5fd79256c246ad/repairnator.json#L9

As you can see, there is the voice “java.lang.AssertionError” together with the voice “skipped”, but there are no other details about the type of the exception. Thus, I don’t know if this voice is associated with a specific exception or not.

File: README.md (hunk 2 above)
Comment (dginelli) [reply README.md:337]: Yes, there are also builds with 0 failing test cases (they are 5.024) and builds with 0 erroring test cases (they are 7.085), but I have not put these values in the two tables.

**Comment by monperrus** [comment]
OK, thanks. Could you put the values in both tables?

**Comment by monperrus** [comment]
For this, we would have to look in the code of Repairnator to see what it means.

**Comment by monperrus** [comment]
ready to merge?

**Comment by dginelli** [comment]
> ready to merge?

If we don't have to put the data about Maven modules and pom.xml files., I think the pull request can be merged.

**Comment by monperrus** [comment]
Thanks a lot! We'll continue to evolve this page in subsequent PRs :-)

**Pull request merged**